packages/backend/src/middleware/cache/data-stores/redis.test.ts

```diff
@@ -13,12 +13,12 @@ describe('Redis Data Store', (): void => {
   const dataStore = createRedisDataStore(redis, ttlMs)
 
   afterEach(async () => {
+    jest.useRealTimers()
     await redis.flushall()
   })
 
   afterAll(async () => {
-    redis.disconnect()
-    jest.useRealTimers()
+    await redis.quit()
```

Comment:
this seems to be the preferred way to stop redis connections: https://github.com/redis/ioredis/issues/1088

Reply:
What about resetting the timers?

Reply:
Moved it to the `afterEach`